src/applications/ivc-champva/10-7959a/tests/unit/containers/ConfirmationPage.unit.spec.js

```diff
@@ -73,7 +73,7 @@ describe('Confirmation page', () => {
 
   it('should render name without suffix if none present', () => {
     const tmpStore = storeBaseTruncated;
-    delete tmpStore.form.data.fullName.suffix;
+    delete tmpStore.form.data.applicantName.suffix;
```

Comment:
`fullName` used to be present in the `storeBaseTruncated` object, but was replaced by `applicantName` in this PR - for some reason the test was [passing previously](https://github.com/department-of-veterans-affairs/vets-website/pull/31505) despite trying to access a nonexistent property.